src/containerapp/azext_containerapp/custom.py

```diff
@@ -516,6 +517,9 @@ def create_containerapp(cmd,
         if scale_rule_http_concurrency:
             if scale_rule_type in ('http', 'tcp'):
                 curr_metadata["concurrentRequests"] = str(scale_rule_http_concurrency)
+        if scale_rule_tcp_concurrency:
+            if scale_rule_type in ('http', 'tcp'):
+                curr_metadata["concurrentRequests"] = str(scale_rule_tcp_concurrency)
```

Comment:
1. Why can parameter `--scale-rule-tcp-concurrency` be passed in as `scale_rule_type==http`?
2. When both `--scale-rule-tcp-concurrency` and `--scale-rule-http-concurrency` parameters are passed in simultaneously, `--scale-rule-tcp-concurrency` will overwrite `--scale-rule-http-concurrency`. Is this expected?

Reply:
If `scale_rule_http_concurrency` and `scale_rule_tcp_concurrency` are essentially the same property on server side, it is recommended to use a more general new parameter `--scale-rule-concurrency` and deprecate the original parameter `--scale-rule-http-concurrency`. Otherwise, using two different parameters to set the same property can easily confuse customers

Reply:
I'm going to close this issue and someone else can open up a new one to deal with it.
I was hoping it was a simple text change but clearly it's not and it requires someone who understand the code to make the change. @anthonychu maybe you can find someone to work on this because right now the help text is totally unusable for someone (like me) who has an 80 column window.